### PR TITLE
Adds pudge the snake

### PR DESCRIPTION
**Pull request by Goldsteel**
[sprites]

He's pudge. He's a pudge. He's pudgey. He's chubby. He's a snake. Where did he come from? Who knows.

image

image

Pudge comes with idle animation action and will soon have a movement state. Spritework by Sage of Light. Additional animations may occur, though others may be dropped depending. I may have discovered that pudge can't have a gender because this could mean he would breed with other cats which is horrifying.

:cl:
 * rscadd: Adds Pudge the snake as a new pet mob

### CODE REVIEW

**Comment by Kharkov72** [comment]
He's In

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 1]
```diff
@@ -178,3 +178,23 @@ var/list/wizard_snakes = list()
...
 	wizard_snakes -= src
 	..()
+
+/mob/living/simple_animal/cat/snek/pudge
+	name = "Pudge"
+	desc = "He's quite chubby!"
+	icon_state = "pudge"
+	icon_living = "pudge"
+	icon_dead = "pudge_dead"
+	gender = NEUTER
```
Comment (DrSnips) [inline code/modules/mob/living/simple_animal/friendly/cat.dm:188]: Should match it's desc of "HE is quite chubby."

**Comment by SageOfLight** [comment]
Tsuchinoko real.

**Comment by DrSnips** [comment]
Squeak(ing) should be added to one of its emotes.
t. Local self proclaimed mythological creature ~~expert~~ fag

**Comment by SageOfLight** [comment]
There's a jump state inside it's icon but there's no conditions under the current mob type to use it, if anyone's wondering.

**Comment by SageOfLight** [comment]
image

**Comment by N3X15** [comment]
There's a point at which a reptile ceases being  a snake and becomes a football.

**Comment by Gazathor** [comment]
> There's a point at which a reptile ceases being a snake and becomes a football.

Tell that to the Japs. they got like a 10,000 dollar bounty on that fat fuck

**Comment by PrimeDSS13** [comment]
The real question is which department deserves him.

**Comment by jknpj** [comment]
Xenobio?

**Pull request merged**